pkg/cmd/kubectlenv/kubectlenv.go

```diff
@@ -28,10 +28,10 @@ func NewCmdKubectlEnv(f util.Factory, ioStreams util.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "kubectl-env",
 		Short: "Generate a script that points KUBECONFIG to the targeted cluster for the specified shell",
-		Long: `Generate a script that points KUBECONFIG to the targeted cluster for the specified shell.
-See each sub-command's help for details on how to use the generated script.
+		Long: `Generate a script that points KUBECONFIG to the currently targeted shoot, seed, or garden cluster for the specified shell.
+To apply this setting automatically in every shell session, consider adding the generated script to your shell's startup configuration.
 
-The generated script points the KUBECONFIG environment variable to the currently targeted shoot, seed or garden cluster.
+See each sub-command's help for details on how to use the generated script.
```

Comment:
Oh I just noticed we have an inconsistency here:
1. "consider adding the generated script to your shell's startup configuration"
2. "consider adding the command at the end of your ... file"
I would recommend to add the command instead of adding the generated script. However, maybe the following is a better `Long` help text for the parent command, WDYT?

```go
Generate a script that points KUBECONFIG to the currently targeted shoot, seed, or garden cluster for the specified shell.

Each sub-command produces a shell-specific script. For details on how to use the printed shell script, such as applying it temporarily to your current session or permanently through your shell’s startup file, refer to the corresponding sub-command’s help.
```

Reply:
I updated the long text of `gardenctl kubectl-env` via: [`5c5804a` (#490)](https://github.com/gardener/gardenctl-v2/pull/490/commits/5c5804afda30360e751aab4822ffb1924c0eb9f2).
I agree that it is the better description and avoids the inconsistency :) 

I've left the sub-command text unchanged i.e. it still suggests adding the **command** to your shell startup script. I hope I got you correctly :)